### ISSUE TITLE
建议修改默认配置

### ISSUE THREAD

**Issue by Old-stan** [elliot-bia/nessus#56]
**描述问题**
Nessus中，有些配置没必要开启，建议直接修改出厂默认配置

**涉及的配置项如下**
- safe_checks
- auto_update_delay
- auto_update
- auto_update_ui
- report_crashes

**截图**
![1](https://github.com/elliot-bia/nessus/assets/27675433/4fdb07f9-4200-4154-be78-80e88c955ec3)

![2](https://github.com/elliot-bia/nessus/assets/27675433/bdf56e99-9196-4f2e-87c0-b02453ce945d)

**Comment by elliot-bia** [comment]
😯这些配置有cli修改命令吗

**Comment by Lesterol** [comment]
Hello. I will support the issue and add that it would be nice to also disable the following parameter by default.
![image](https://github.com/elliot-bia/nessus/assets/94538079/55f946e9-689d-4ff1-bfd4-8a7f4a925ad6)
As for nessuscli, you can disable this option with the command: `nessuscli fix --set send_telemetry=no`
The commands above are also disabled by analogy:
`nessuscli fix --set auto_update_delay=no`
`nessuscli fix --set auto_update=no`
`nessuscli fix --set auto_update_ui=no`
`nessuscli fix --set report_crashes=no`
After applying these settings, nessus reboots and the crack disappears. So if you can fix this and set these default options to "no", then that would be great.

**Comment by elliot-bia** [comment]
Copy that, it will be fix in the few days

**Comment by Lesterol** [comment]
> Copy that, it will be fix in the few days

Hello Elliot. Have you tried to fix this case yet?

**Comment by elliot-bia** [comment]
No...It's Chinese New Year these days, everyone is busy for preparing.
Maybe this Saturday....

**Comment by elliot-bia** [comment]
I got free time now, it will be release in the next two weeks!
